doc/demo-beta.1/README.md

```diff
@@ -0,0 +1,218 @@
+LNP-node demo
+===
+
+### Introduction
+
+This document contains demonstration of `lnp-node` functionality as for [version 0.1.0-beta.1](https://github.com/LNP-BP/lnp-node/releases/tag/v0.1.0-beta.1) and mainly follows [this video-recorded demo]( https://www.youtube.com/watch?v=ns58fUlyd5o).
+
+Two different setups are available:
+- [local installation](#local)
+- [docker](#docker)
+
+Once either of them is complete, you can proceed with the actual [demo](#demo)
+
+## Local
+
+This setup consists in a local installation of `lnp-node`.
+
+#### Requirements
+
+- [cargo](https://doc.rust-lang.org/book/ch01-01-installation.html#installation)
+- [git](https://git-scm.com/downloads)
+
+Furthermore, you will need to install a number of system dependencies:
+```bash=
+sudo apt install -y build-essential pkg-config libzmq3-dev libssl-dev libpq-dev libsqlite3-dev cmake
+```
+### Setup
+
+Alongside `lnp-node`, we also need to install `rgb-node` which will be queried for token-related data, then we can launch it in the first terminal (ctrl+C to stop it):
+```bash=
+git clone https://github.com/LNP-BP/lnp-node
+cd lnp-node/doc/demo-beta.1
+cargo install --locked --root . rgb_node --version 0.2.0-beta.4 --all-features
+cargo install --locked --root . --git "https://github.com/LNP-BP/lnp-node#0.1.0-beta.1" --all-features
```

Comment:
```suggestion
cargo install rgb_node --root . --locked --version 0.2.0-beta.4 --all-features
cargo install lnp_node --root . --locked --version 0.1.0-beta.1 --all-features
```

Reply:
There is a long-standing issue (#28) I wasn't able to figure out with crates installation, installation from git is the easiest workaround I found. Possibly this will be solved in the next tag anyways.